packages/eslint-plugin/docs/rules/no-empty-function.md

```diff
@@ -28,7 +28,8 @@ This rule adds the following options:
 type AdditionalAllowOptionEntries =
   | 'private-constructors'
   | 'protected-constructors'
-  | 'decoratedFunctions';
+  | 'decoratedFunctions'
+  | 'overrideMethods';
```

Comment:
Question before merge: should we go with camel case or kebab case? ESLint always uses kebab case, and so are the two other options here.

Reply:
i don't think it hugely matters - I think we use a mix of everything in this codebase.
we can always write a lint rule and standardise later!